cli/config/config.go

```diff
@@ -0,0 +1,97 @@
+package config
```

Comment:
See these PRs, they add logic for CLI config:

- https://github.com/rilldata/rill-developer/pull/1839
- https://github.com/rilldata/rill-developer/pull/1852